src/main/java/org/opensearch/knn/index/KNNSettings.java

```diff
@@ -86,7 +86,6 @@ public class KNNSettings {
      */
     public static final boolean KNN_DEFAULT_FAISS_AVX2_DISABLED_VALUE = false;
     public static final String INDEX_KNN_DEFAULT_SPACE_TYPE = "l2";
-    public static final String INDEX_KNN_DEFAULT_SPACE_TYPE_FOR_BINARY = "hammingbit";
```

Comment:
why this is removed?

Reply:
is the [LegacyFieldMapper.java](https://github.com/opensearch-project/k-NN/pull/1854/files#diff-8e86283d3d21679ed977858ff1b613bc833e681df86ac48afcf97de9735d7cdf) the only place where this has been used?

Reply:
Yes. This value is not coming from Setting so I just used space type directly in
https://github.com/opensearch-project/k-NN/pull/1854/files#diff-8e86283d3d21679ed977858ff1b613bc833e681df86ac48afcf97de9735d7cdfR87

Reply:
Hmm. Actually, Having this might make sense to have a consistency...

Reply:
Let me revert it.